package.json

```diff
@@ -1,25 +1,64 @@
 {
   "name": "linter-htmlhint",
-  "providedServices": {
-    "linter": {
-      "versions": {
-        "1.0.0": "provideLinter"
-      }
-    }
-  },
-  "main": "./lib/init",
```

Comment:
It is, but almost all the other projects are setup that way, and the `build-package.sh` script for the Travis-CI config in atom/ci automaticlaly runs any configured linters if the code is within a `lib` folder. ([see here](https://github.com/atom/ci/blob/master/build-package.sh#L76))